lib/ask-stack-result-view.coffee

```diff
@@ -198,7 +215,17 @@ class AskStackResultView extends ScrollView
           atom.workspace.activatePreviousPane()
           # editor = atom.workspace.activePaneItem
           editor = atom.workspace.getActivePaneItem()
-          editor.insertText(code)
+          if qa == true
+            editor.insertText("Insert from Stack Overflow", {select: true})
+            editor.toggleLineCommentsInSelection();
+            editor.insertNewlineBelow();
+            editor.insertText("Author: #{author_name} - #{author_src}", {select: true})
+            editor.toggleLineCommentsInSelection();
```

Comment:
Cool. I've asked for support in the forums as well before and some people were able to let. Let me know how it goes.